source/rtp_api.c

```diff
@@ -67,14 +67,14 @@
 
 /*-----------------------------------------------------------*/
 
-static size_t CalculateSerializedPacketLength( const RtpPacket_t * pRtpPacket );
+static uint32_t CalculateSerializedPacketLength( const RtpPacket_t * pRtpPacket );
```

Comment:
If we want to change the type from size_t to uint32_t, should we apply this to all other repos?

Reply:
Yes we can discuss this.